index.md

```diff
@@ -136,7 +136,7 @@ nav: index
             <div class="bg-color"></div>
           </div>
           <div class="address">
-            <h4><strong>CHENNAI TRADE CENTER</strong></h4>
+            <h4><a href = "https://www.google.com/maps/place/Chennai+Trade+Centre/@13.013676,80.1878456,17z/data=!4m5!3m4!1s0x3a526734ffcb019d:0xe47e2c5b27491350!8m2!3d13.013676!4d80.1900343"><strong>CHENNAI TRADE CENTER</strong></a></h4>
             <p>OCTOBER 12, 13</p>
             <span>CTC Complex,<br /> Mount Poonamalle High Rd, Nandambakkam, Chennai, Tamil Nadu 600089</span>
             <!-- <div style="position: relative;">
```

Comment:
Modify the below link instead.

Reply:
This is the map link to address. Do you want me to short the link?

Reply:
No, uncomment the div and update the a tag present there.

Reply:
Okay noted! Did.